templates/travis/.travis/check_pulpcore_imports.sh.j2

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+{% include 'header.j2' %}
+
+set -uv
+
+# check for imports not from pulpcore.plugin. exclude tests
+MATCHES=$(grep -r --include \*.py "from pulpcore.*import" . | grep -v "tests\|plugin")
```

Comment:
I suspect this command to present all findings collapsed in a single line.
If you add a `-n` to the first `grep`, it should report the line-number next to the filename, where it found the line.

Reply:
Will add `-n`. It does seem to report $MATCHES on newlines when I use echo double quotes:

```
ERROR: Detected bad imports from pulpcore:
./pulp_rpm/app/modulemd.py:4:from pulpcore.app.models.content import Artifact
./pulp_rpm/app/modelresource.py:1:from pulpcore.app.modelresource import QueryModelResource

Plugins should import from pulpcore.plugin.                                                                                                                                                                                    
```